tests/utils/test_docs.py

```diff
@@ -34,14 +35,20 @@ class TestGetDocsUrl(unittest.TestCase):
             ),
             (
                 '2.0.0.dev0',
-                'migration.html',
+                'tutorial.html',
                 'http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/'
-                'apache-airflow/latest/migration.html',
+                'apache-airflow/latest/tutorial.html',
+            ),
+            ('1.10.10', None, 'https://airflow.apache.org/docs/apache-airflow/1.10.10/'),
+            (
+                '1.10.10',
+                'tutorial.html',
+                'https://airflow.apache.org/docs/apache-airflow/1.10.10/tutorial.html',
             ),
-            ('1.10.0', None, 'https://airflow.apache.org/docs/1.10.0/'),
-            ('1.10.0', 'migration.html', 'https://airflow.apache.org/docs/1.10.0/migration.html'),
         ]
     )
-    def test_should_return_link(self, version, page, expected_urk):
+    def test_should_return_link(self, version, page, expected_url):
         with mock.patch('airflow.version.version', version):
-            self.assertEqual(expected_urk, get_docs_url(page))
+            self.assertEqual(expected_url, get_docs_url(page))
+            resp = requests.get(expected_url)
+            assert resp.status_code == 200
```

Comment:
Not sure about sending requests (in general it's not best practice) but I don't know other way to make usre what we use works - without this the test gives false positives.

Reply:
It seems fine in this case, but we can speed up this code a bit if we use HEAD requests.